src/Camera.cpp

```diff
@@ -190,15 +190,17 @@ void Camera::initializeCalibration() {
   for (int i = 0; i < nb_board_est; i++) {
     std::shared_ptr<BoardObs> board_obs_temp =
         board_observations_[indbv[shuffled_board_ind[i]]].lock();
-    img_points.push_back(board_obs_temp->pts_2d_);
-    std::vector<int> corners_idx_temp = board_obs_temp->charuco_id_;
-    std::vector<cv::Point3f> pts_3d_temp;
-    std::shared_ptr<Board> board_3d_ptr = board_obs_temp->board_3d_.lock();
-    if (board_3d_ptr) {
-      for (int k = 0; k < corners_idx_temp.size(); k++)
-        pts_3d_temp.push_back(board_3d_ptr->pts_3d_[corners_idx_temp[k]]);
+    if (board_obs_temp) {
```

Comment:
I do not get this one :s

Reply:
Same as in the other comment. Just checking whether the pointer exists before using it.